charts/launch-agent/templates/rbac.yaml

```diff
@@ -1,17 +1,19 @@
+---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: wandb-launch-serviceaccount
-  namespace: wandb
+  namespace: {{ .Values.namespace }}
```

Comment:
I'm not we can support installing in another namespace, I believe that the `wandb` namespace is hard coded in a few places in the sdk